### ISSUE TITLE
Upgrade from 6.1.1 to 6.1.2 breaks Gradle

### ISSUE THREAD

**Issue by mirabilos** [dependency-check/dependency-check-gradle#216]
I’m currently upgrading a Maven and a Gradle project from 6.1.1 to 6.1.2, and while the Maven project works, the Gradle project stops working:

```
$ ./gradlew clean

FAILURE: Build failed with an exception.

* Where:
Build file '/home/tglase/Misc/Vendor/ECN-Bits/android/app/build.gradle' line: 1

* What went wrong:
A problem occurred evaluating project ':app'.
> Failed to apply plugin 'com.android.internal.application'.
   > class com.google.common.base.Suppliers$NonSerializableMemoizingSupplier cannot be cast to class java.util.function.Supplier (com.google.common.base.Suppliers$NonSerializableMemoizingSupplier is in unnamed module of loader org.gradle.internal.classloader.VisitableURLClassLoader @2afae768; java.util.function.Supplier is in module java.base of loader 'bootstrap')

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s
```

This error message is [found somewhere else](https://github.com/GoogleCloudPlatform/artifact-registry-maven-tools/issues/27) but the solution shown there (adding an explicit Guava dependency) does not work here.

You can reproduce that in the following project: https://github.com/tarent/ECN-Bits/tree/master/android

**Comment by OrangeDog** [comment]
Also had a Guava-related problem going from 6.1.1 to 6.1.2, but a different exception:

> Unable to find method ''java.util.stream.Collector com.google.common.collect.ImmutableList.toImmutableList()''
> 'java.util.stream.Collector com.google.common.collect.ImmutableList.toImmutableList()'

**Comment by OrangeDog** [comment]
Same problem in 6.1.3

**Comment by mirabilos** [comment]
Seems to be fixed in 6.1.5 which I cannot even find on github…

**Comment by jeremylong** [comment]
Sorry - I forgot to push the tag: https://github.com/dependency-check/dependency-check-gradle/tree/v6.1.5
